package.json

```diff
@@ -11,6 +11,10 @@
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook"
   },
+  "engines": {
+    "yarn": "1.x",
```

Comment:
Could you explain to me why this is important?

Reply:
> Could you explain to me why this is important?

this is to inform Heroku the versions, it will be using